en/mkdocs.yml

```diff
@@ -291,8 +291,22 @@ nav:
       - 'Configure Provisioning - To Remove':
         - 'Overview': guides/identity-provisioning/provisioning-overview.md
   - 'Extend':
-    - 'Authentication Script API': apis/authentication-script-reference.md
-    - 'Configuration Catalog': apis/configuration-catalog.md
+    - 'Authentication Script API': extend/authentication-script-reference.md
```

Comment:
Shall we change these headings with verb-like beginnings so that it is in sync with the guides and other docs?

Reply:
When referring to other apis in the "APIs" section, we have not used a verb in the beginning. We could change them all to "Use xxxx API" but I personally think this format is fine for APIs. WDYT?